build/nuget-release.yml

```diff
@@ -6,6 +6,9 @@ pr: none
 parameters:
   - name: 'Package.Version'
     type: 'string'
+  - name: 'Package.Name'
+    type: 'string'
+    default: '*'
```

Comment:
In what cases do we want that ?

Reply:
The cases you proposed 😄 . If you want to update a single package without the other packages to update.

Reply:
To be honest, I can't remember that.
And since it seems we do not have an issue for this on GH and the fact that we would run into some other issues as you mention here, I think we should just abandon this idea.